cromshell

```diff
@@ -72,6 +87,20 @@ TERMINAL_STATES="Succeeded Failed Aborted"
 
 ################################################################################
 
+function doomedLogo() 
+{
+	# Figure out which decoder to use:
+	base64 -D <<<"MQo=" &> /dev/null
```

Comment:
No - this block of code is to identify which base64 is in use in the system.  Depending on which one is available (gnu vs bsd vs ???) the flags are different and I need to call base64 differently to unpack the special logo.

I picked that b64 string because it's a single encoded character.